src/main/java/com/watch/shop/app/model/entity/QuartzWatch.java

```diff
@@ -0,0 +1,12 @@
+package com.watch.shop.app.model.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder(toBuilder = true)
+@EqualsAndHashCode(callSuper = true)
+public class QuartzWatch extends Watch {
+    private boolean luminousHands;
```

Comment:
please check Java naming convention for boolean. THere are exception, but 99% cases - is... or has...